### PR TITLE
feat: user register and JWT/token endpoints

### LINKED ISSUE #72: Complete create-token endpoint
The VERIFIED ADMIN will generate a string of numbers (5 digits) for a user(s) to join the congregation.

- [x] Create the create-token endpoint.
- [ ] Write integration tests that cover creation of the token, and verification of the token.

### PR DESCRIPTION
**Pull request by JudeDavis1**
At the moment the surrounding logic is unfinished. The endpoint logic is written, however, we need to do some things before using it.

We need to:
- Bind the admin user (who is creating the congregation) to the congregation just as it is being created.
- Complete integration tests for token verification (relies on the above).
- Use our RBAC discussion to create JWTs and use those for some more of our endpoints.
- Use those JWTs (at the USER level) to get their corresponding `UserID` to verify tokens.

### CODE REVIEW

**Comment by JudeDavis1** [comment]
forgive the commented blocks

**Comment by JudeDavis1** [comment]
They are mainly for retrieving the user ID from JWTs, but since we don't _really_ have JWT setting logic then we will only use those later else we get 400 errors all over

**Comment by JudeDavis1** [comment]
But maybe don't merge just yet, might do more here

**Review by j-koziel — COMMENTED** [review]
LGTM 👍

**Comment by j-koziel** [comment]
> LGTM 👍

Forgot to approve 🤦‍♂️

**Review by j-koziel — APPROVED** [review]

**Comment by JudeDavis1** [comment]
I'll keep this open because there's some unfinished stuff here @j-koziel

File: packages/frontend/lib/stores/register-user.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  async (formData, { rejectWithValue }) => {
+    try {
+      await loginUser(formData);
+      toast({
+        title: "Success",
+        description: "Loggined in!",
+        variant: "success",
+      });
+    } catch (error) {
+      console.log("jjdfisd");
```
Comment (j-koziel) [inline packages/frontend/lib/stores/register-user.ts]: you alright there

**Review by j-koziel — APPROVED** [review]

File: packages/frontend/lib/stores/register-user.ts (hunk 1 above)
Comment (JudeDavis1) [reply packages/frontend/lib/stores/register-user.ts]: No 🥲

**Pull request merged**